templates/repo/bare.tmpl

```diff
@@ -21,8 +21,8 @@
                 <div class="panel-content">
                     <div id="repo-clone" class="clear text-center">
                         <h2>{{.i18n.Tr "repo.clone_this_repo"}}</h2>
-                        <button class="btn btn-blue current left btn-left-radius" id="repo-clone-ssh" data-link="{{.CloneLink.SSH}}">SSH</button>
-                        <button class="btn btn-gray left" id="repo-clone-https" data-link="{{.CloneLink.HTTPS}}">HTTPS</button>
+                        <button class="btn btn-blue current left btn-left-radius" {{if $.DisableSSH}}style="visibility: hidden" {{end}} id="repo-clone-ssh" data-link="{{.CloneLink.SSH}}">SSH</button>
```

Comment:
I think adding a ```hidden``` css class would be a better approach instead of using inline CSS.